lldb/bindings/interface/SBSaveCoreOptionsDocstrings.i

```diff
@@ -0,0 +1,72 @@
+%feature("docstring",
+"A container for options to use when saving a core file.
+
+SBSaveCoreOptions includes API's to specify the memory regions and threads to include
+when generating a core file. These options are not exclusive the existing SaveCoreStyle option.
+
+Full will save off all thread and memory regions, ignoring the memory regions and threads in
+the options object.
```

Comment:
```suggestion
eSaveCoreFull will save off all thread and memory regions, ignoring the memory regions and threads in
the options object.
```

Maybe make these bullet points to make it clear that you're iterating over the options, something like this:

```
* eSaveCoreFull: save off all thread and memory regions, ignoring the memory regions and threads specified in this option object. 
```

Reply:
Sweet, I wanted to do a list but didn't realize if the docstrings supported markdown.